docs-content/getting-started/fullstack-frameworks/next-js.md

```diff
@@ -28,6 +28,39 @@ all-in-one.
 yarn add @highlight-run/next @highlight-run/react highlight.run
 ```
 
+## Environment Configuration (optional)
+
+> This section is extra opinionated about Next.js constants. It's not for everyone. We like how `zod` and TypeScript work together to validate `process.env` inputs... but this is a suggestion. Do your own thing!
```

Comment:
maybe we can say that if you don't want to do this, replace `CONSTANTS.` imports and references with your own constants.

Reply:
Updated.